### ISSUE TITLE
Is there an English version of this plugin?

### ISSUE THREAD

**Issue by franklinqin0** [vuepress-reco/vuepress-plugin-kan-ban-niang#1]

**Comment by latteandcat** [comment]
Can you describe what do you want about the English version?

**Comment by franklinqin0** [comment]
The pop-up text.

**Comment by latteandcat** [comment]
wow. the pop-up text is [optional](https://vuepress-theme-reco.recoluan.com/views/plugins/kanbanniang.html#option-api)

**Comment by franklinqin0** [comment]
So you mean I could change the [message](https://vuepress-theme-reco.recoluan.com/views/plugins/kanbanniang.html#messages) to English by myself? Can't there be a default English message?

**Comment by latteandcat** [comment]
Yes, you can write whatever you want in the options,  I will think about "default English message", maybe it will  be added in the next version

**Comment by franklinqin0** [comment]
Nice, thanks a lot~~~

**Comment by latteandcat** [comment]
嘻嘻

**Comment by LocTran016** [comment]
Excuse me, how could I change the pop-up text? I have tried this but doesn't help:
![image](https://user-images.githubusercontent.com/64778285/93197264-9fa75780-f775-11ea-86c1-d79859b1a5f4.png)
Thanks

**Comment by loctran016** [comment]
Excuse me, is there any way to change this button into English then:
![image](https://user-images.githubusercontent.com/64778285/93206444-9c1acd00-f783-11ea-9790-97fbc77db733.png)

**Comment by loctran016** [comment]
Here is the [the blog](https://loctran016.gitlab.io/notice/) and [the source code](https://gitlab.com/LocTran016/notice/) as well

**Comment by loctran016** [comment]
This is seem look like a bug rather than translating from Chinese into English I'd rather posted another post about this

**Comment by latteandcat** [comment]
> This is seem look like a bug rather than translating from Chinese into English I'd rather posted another post about this

get it
